appveyor-build.cmd

```diff
@@ -11,11 +11,20 @@ if exist "%ProgramFiles(x86)%\Microsoft Visual Studio 12.0\common7\ide\devenv.ex
 if exist "%ProgramFiles%\Microsoft Visual Studio 12.0\common7\ide\devenv.exe" set VisualStudioVersion=12.0
 
 :vsversionset
-if '%VisualStudioVersion%' == '' echo Error: Could not find an installation of Visual Studio && goto :eof
+if '%VisualStudioVersion%' == '' echo Error: Could not find an installation of Visual Studio && goto :failure
 
 if exist "%ProgramFiles(x86)%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe" set _msbuildexe="%ProgramFiles(x86)%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe"
 if exist "%ProgramFiles%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe"      set _msbuildexe="%ProgramFiles%\MSBuild\%VisualStudioVersion%\Bin\MSBuild.exe"
-if not exist %_msbuildexe% echo Error: Could not find MSBuild.exe. && goto :eof
+if not exist %_msbuildexe% echo Error: Could not find MSBuild.exe. && goto :failure
+
+:: See <http://www.appveyor.com/docs/environment-variables>
+if defined APPVEYOR (
+    rem See <http://www.appveyor.com/docs/build-phase>
+    if exist "C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" (
+	rem HACK HACK HACK
```

Comment:
Well, I only called it a hack because the name `_msbuildexe` implies that it just holds the path to msbuild's executable, so adding a parameter to it seems ... a bit off.